src/ralph/configuration_management/views.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.urlresolvers import NoReverseMatch
+from django.utils.safestring import mark_safe
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.admin.filters import custom_title_filter
+from ralph.admin.helpers import get_admin_url
+from ralph.admin.views.extra import RalphDetailView
+from ralph.configuration_management.models import SCMCheckResult
+# NOTE(romcheg): These functions could be moved to a common place
+from ralph.security.views import _linkify, _url_name_for_change_view
+
+
+logger = logging.getLogger(__name__)
+
+
+class SCMCheckInfo(RalphDetailView):
+
+    icon = 'cogs'
+    label = 'SCM info'
+    name = 'scm_info'
+    url_name = None
+    template_name = 'configuration_management/scminfo/scm_info.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        try:
+            check = self.object.scmstatuscheck
+        except ObjectDoesNotExist:
+            check = None
+        context['scm_check'] = check
+
+        hostname = getattr(self.object, 'hostname', None)
+
+        if hostname:
+            context['scm_tool_url'] = settings.SCM_TOOL_URL.format(
+                hostname=hostname
+            )
+
+        return context
+
+
+class SCMStatusCheckInChangeListMixin(object):
+
+    icon_no_scan = '-'
+    icon_scan_ok = '<i class="fa fa-check-circle" aria-hidden="true"></i>'
```

Comment:
dj.choices has a "little feature" called extra - https://github.com/ambv/dj.choices/#extra-attributes-on-choices - consider using it for classes for different icons (`fa-check-circle` etc)